### ISSUE TITLE
Repository name change?

### ISSUE THREAD

**Issue by rouson** [OpenFortranProject/ofp-sdf#12]
To avoid any confusion, it would be great to change the name of the other [open-fortran-parser repository](https://github.com/OpenFortranProject/open-fortran-parser) to something like "legacy-ofp" or "legacy-open-fortran-parser" and then change the name of the current repository to something like "open-fortran-parser" or "ofp". Because the number of downloads and installations is likely to rise when OpenCoarrays 2.0.0 releases with support for source transformation, which will likely happen after an OFP 1.0.0 release, it would be nice to make the name change before the OFP 1.0.0 release.

**Comment by mbdevpl** [comment]
@rouson Yes, by all means, please change the name(s). This weekend I was experimenting with the code from legacy repository... However the fact that it's legacy is not mentioned in any visible place. I even submitted a trivial pull request there ;)
